tests/solana_tests/primitives.rs

```diff
@@ -1275,7 +1261,7 @@ fn test_overflow_detect_unsigned() {
             // Calculate a number that when multiplied by first_operand_rand, the result will overflow N bits
             let second_operand_rand = rng.gen_biguint_range(&BigUint::from(2usize), &limit);
 
-            let res = contract
+            let _ = contract
```

Comment:
Does this and other lines need `let _ = `?

Reply:
Removed.